docs/examples/Transformer Examples.ipynb

```diff
@@ -0,0 +1,563 @@
+    "sym_load[\"p_specified\"] = [1e3, 1e2]\n",
+    "sym_load[\"q_specified\"] = [5e3, 1e3]\n",
+    "\n",
+    "# source\n",
+    "source = initialize_array(\"input\", \"source\", 1)\n",
+    "source[\"id\"] = [1]\n",
+    "source[\"node\"] = [2]\n",
+    "source[\"status\"] = [1]\n",
+    "source[\"u_ref\"] = [1.0]\n",
+    "\n",
+    "# three-winding transformer\n",
+    "three_winding_transformer[\"id\"] = [3]\n",
+    "three_winding_transformer[\"node_1\"] = [2]\n",
+    "three_winding_transformer[\"node_2\"] = [4]\n",
+    "three_winding_transformer[\"node_3\"] = [6]\n",
+    "three_winding_transformer[\"status_1\"] = [1]\n",
+    "three_winding_transformer[\"status_2\"] = [1]\n",
+    "three_winding_transformer[\"status_3\"] = [1]\n",
+    "three_winding_transformer[\"sn_1\"] = [6.0e7]\n",
+    "three_winding_transformer[\"sn_2\"] = [5.0e7]\n",
+    "three_winding_transformer[\"sn_3\"] = [1.0e7]\n",
+    "three_winding_transformer[\"uk_12\"] = [0.09]\n",
+    "three_winding_transformer[\"uk_13\"] = [0.06]\n",
+    "three_winding_transformer[\"uk_23\"] = [0.03]\n",
+    "three_winding_transformer[\"pk_12\"] = [5.0e4]\n",
+    "three_winding_transformer[\"pk_13\"] = [1.0e4]\n",
+    "three_winding_transformer[\"pk_23\"] = [8.0e3]\n",
+    "three_winding_transformer[\"i0\"] = [0]\n",
+    "three_winding_transformer[\"p0\"] = [0]\n",
+    "three_winding_transformer[\"winding_1\"] = [2]\n",
+    "three_winding_transformer[\"winding_2\"] = [1]\n",
+    "three_winding_transformer[\"winding_3\"] = [1]\n",
+    "three_winding_transformer[\"clock_12\"] = [11]\n",
+    "three_winding_transformer[\"clock_13\"] = [11]\n",
+    "three_winding_transformer[\"tap_side\"] = [0]\n",
+    "three_winding_transformer[\"tap_pos\"] = [0]\n",
+    "three_winding_transformer[\"tap_min\"] = [-10]\n",
+    "three_winding_transformer[\"tap_max\"] = [10]\n",
+    "three_winding_transformer[\"tap_nom\"] = [0]\n",
+    "three_winding_transformer[\"tap_size\"] = [1380]\n",
+    "\n",
+    "# all\n",
+    "input_data3 = {\n",
+    "    \"node\": node,\n",
+    "    \"three_winding_transformer\": three_winding_transformer,\n",
+    "    \"sym_load\": sym_load,\n",
+    "    \"source\": source\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c25444ec",
+   "metadata": {},
+   "source": [
+    "### One-time power flow calculation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 173,
+   "id": "b4e63620",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "------node result------\n",
+      "   id  energized      u_pu             u   u_angle             p             q\n",
+      "0   2          1  0.257969  35599.780770  0.061612  3.193357e+07  1.915647e+09\n",
+      "1   4          1  0.162156  11188.754086  0.579654 -1.000000e+03 -5.000000e+03\n",
+      "2   6          1  0.000001      0.018674  0.502210 -9.999985e+01 -1.000000e+03\n"
```

Comment:
the `u_pu` and `u` are **extremely** low. please choose different settings to make all `u_pu`s more reasonable (0.16 for node id 4 is already quite small, but 1e-6 for node id 6 is a bit extreme)